File: send_telegram.rb
require "open-uri"

class PagesController < ApplicationController

  def send_telegram(message)
      token = "YOUR_TOKEN"
      
      data = {
        text: message,
        chat_id: "YOUR_CHAT_ID"
      }
      url = "https://api.telegram.org/bot#{token}/sendMessage?#{data.to_param}&parse_mode=html"
      puts "telegram url: " + url
      open(url).read
    end
    
    send_telegram("hi!")
end

File: send_telegram.php
<?php
function send_telegram($message) {
	$token = "YOUR_TOKEN";
	$data = [
		'text' => $message,
		'chat_id' => 'YOUR_CHAT_ID'
	];

	file_get_contents("https://api.telegram.org/bot$token/sendMessage?" . http_build_query($data) . "&parse_mode=html");
}

send_telegram("hi!")
